src/Features/Core/Portable/IncrementalCaches/SymbolTreeInfoIncrementalAnalyzerProvider.cs

```diff
@@ -165,13 +165,6 @@ public override Task AnalyzeDocumentAsync(Document document, SyntaxNode bodyOpt,
                     return SpecializedTasks.EmptyTask;
                 }
 
-                if (bodyOpt != null)
-                {
-                    // This was a method level edit.  This can't change the symbol tree info
-                    // for this project.  Bail immediately.
-                    return SpecializedTasks.EmptyTask;
```

Comment:
this check was unfortunately invalid.  While it is true that the SymbolTreeInfo isn't different, the checksum for the project will be different, so we won't be able to reuse it.